architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -76,10 +76,12 @@ documentation] for more information on node management.
 
 As an OpenShift administrator, you can
 link:../../admin_guide/manage_nodes.html[manage nodes] in your instance using
```

Comment:
I don't think we have that same assumption that the reader is definitely an admin in the Arch topics? I did update it slightly though:

"Administrators can manage nodes in an OpenShift instance using the CLI."